cg/apps/sequencing_metrics_parser/parsers/bcl_convert.py

```diff
@@ -149,16 +159,36 @@ def get_flow_cell_name(self) -> str:
 
     def get_q30_bases_percent_for_sample_in_lane(self, sample_internal_id: str, lane: int) -> float:
         """Return the percent of bases that are Q30 for a sample and lane."""
-        metric: BclConvertQualityMetrics = self.get_metrics_for_sample_and_lane(
-            metrics_list=self.quality_metrics, sample_internal_id=sample_internal_id, lane=lane
+        metric: List[BclConvertQualityMetrics] = self.get_read_pair_metrics_for_sample_and_lane(
+            sample_internal_id=sample_internal_id, lane=lane
         )
-        return metric.q30_bases_percent
+        return self.calculate_read_pair_q30_bases_percent(metric_list=metric)
+
+    @classmethod
+    def calculate_read_pair_q30_bases_percent(
+        cls, metric_list: List[BclConvertQualityMetrics]
+    ) -> float:
+        """Calculate the percent of bases that are Q30 for read pairs."""
+        mean_read_pair_q30_bases_percent: float = 0
+        for metric in metric_list:
+            mean_read_pair_q30_bases_percent += metric.q30_bases_percent
+        return round(mean_read_pair_q30_bases_percent / 2, 2)
+
+    @classmethod
+    def calculate_mean_quality_score(cls, metric_list: List[BclConvertQualityMetrics]) -> float:
+        """Calculate the mean quality score for a list of metrics."""
+        total_q_score: float = 0
+        for metric in metric_list:
+            total_q_score += metric.mean_quality_score_q30
+        return round(total_q_score, 2)
 
-    def get_mean_quality_score_fot_sample_in_lane(
+    def get_mean_quality_score_for_sample_in_lane(
         self, sample_internal_id: str, lane: int
     ) -> float:
         """Return the mean quality score for a sample and lane."""
-        metric: BclConvertQualityMetrics = self.get_metrics_for_sample_and_lane(
-            metrics_list=self.quality_metrics, sample_internal_id=sample_internal_id, lane=lane
+        metric_list: List[
```

Comment:
Isn't this similar to suffixes such as `_obj`? I think Henrik has suggested plurals for lists, so in this case `metrics` instead of `metric_list`